ebos/eclwriter.hh

```diff
@@ -39,8 +39,22 @@
 
 #include <dune/grid/common/partitionset.hh>
 
-#include <limits>
-#include <stdexcept>
+#if OPM_HAVE_DAMARIS
+#include <Damaris.h>
+#endif
+
+#if OPM_HAVE_DAMARIS
+#include <Damaris.h>
+#endif
+
+#if OPM_HAVE_DAMARIS
+#include <Damaris.h>
+#endif
+
+#if HAVE_DAMARIS
+#include <Damaris.h>
+#endif
```

Comment:
i'd like to see this a fifth time so we are REALLY sure ;P

and don't remove the other includes.